otcextensions/sdk/dns/v2/_base.py

```diff
@@ -88,11 +88,11 @@ def _get_next_link(cls, uri, response, data, marker, limit, total_yielded):
         # This prevents duplication of query parameters that with large
         # number of pages result in HTTP 414 error eventually.
         if next_link:
```

Comment:
overall I think we can base from sdk _base: _get_next_link is already defined there. btw, do we still need _base class if we switch to sdk resources?

Reply:
fixed